traitsui/qt4/range_editor.py

```diff
@@ -739,7 +749,7 @@ def update_object(self):
             else:
                 col = OKColor
 
-            self.value = value  
+            self.value = value
```

Comment:
I didn't remember touching this line, but remembered I recently updated VSCode settings to kill trailing whitespace whenever I save a file 😄